app/models/stolen_record.rb

```diff
@@ -51,6 +51,10 @@ def self.revised_date_format
     '%a %b %d %Y'
   end
 
+  def self.iso8601_date_format
```

Comment:
Not yet. I saw the revised_date_format method and thought it was worth adding. I'll nuke it if I don't end up using it.